packages/components/src/components/hds/table/th-selectable.hbs

```diff
@@ -20,7 +20,6 @@
       {{on "change" this.onSelectionChange}}
     />
     {{#if this.isSortable}}
-      <span id={{this._labelId}} class="sr-only">selection state</span>
```

Comment:
[thought/nit] _technically_, this should also be added to the changelog (same for the other places where you have removed the `sr-only` span elements)